variables.tf

```diff
@@ -2,10 +2,18 @@ variable "app_name" {
   type        = string
   description = "Scheduled Fargate Application name."
 }
-variable "ecs_cluster_name" {
-  type        = string
-  description = "ECS Cluster name to host the scheduled fargate task. Defaults to creating its own cluster."
-  default     = null
+variable "existing_ecs_cluster" {
```

Comment:
Is `use_existing` something that could just be implied by `existing_ecs_cluster` not being null